packages/gatsby/src/redux/types.ts

```diff
@@ -1,7 +1,23 @@
+export enum ProgramStatus {
+  BOOTSTRAP_FINISHED = `BOOTSTRAP_FINISHED`,
+  BOOTSTRAP_QUERY_RUNNING_FINISHED = `BOOTSTRAP_QUERY_RUNNING_FINISHED`,
+}
+
 export interface IReduxState {
-  nodes?: any // TODO
-  nodesByType: Map<any, any> // TODO
+  status: ProgramStatus
+  nodes?: {
```

Comment:
If you extract this into an interface, you can then easily type nodesByType below

Reply:
sounds great, i'll follow that up in a future PR. I didn't fully understand yet what `nodesByType` was